Code-the-Dream-School/hh-team4-back
# 30 Random medications added to SeedDatabase

## Description

In the Seed Database I added a total of 30 random medications to the array.

## Related Issue

Not enough Data for pagination and proper testing.

## Type of Changes

increase data pool

## Testing Instructions
Same as before with Postman, just run get all medications route and you will see them all.
_In a list format, tell us how to test this PR._

## Learnings (Optional)

_Document any things you learned or 'gotchas' you experienced._
The only gotcha I noticed, is every time we run the seed() it wipes users too, So you will need to signup/login again, because I probably just wiped your user info when i re-seeded.
